BUILDING-configure.md

```diff
@@ -1,29 +1,64 @@
 Building using `configure`
 ==========================
 
-The build requires `libpq`, the C client library for PostgreSQL.  This library
-must be installed before you can build libpqxx.  You'll need the headers as
-well as the library binary.
-
 The instructions will assume that you're working from a command-line shell.
 If you prefer to work from an IDE, you'll have to know how your IDE likes to
 do things, and you'll want to follow the shell instructions as a guide.
 
+Prerequisites
+-------------
+
+Before you can begin the build process for libpqxx, you must first install
+`libpq-dev` and the latest `libpq5` library. These are C client libraries
+for PostgreSQL. 
+
+Enter this terminal command to obtain the latest `libpq5` library.
+
+```shell
+    apt-cache policy libpq5
+```
+
+From the output, select the `Candidate` value.
+
+For example, on Linux (Ubuntu 22.04) the terminal output is as follows:
+
+```shell
+libpq5:
+  Installed: 14.7-0ubuntu0.22.04.1
+  Candidate: 14.7-0ubuntu0.22.04.1
+  Version table:
+ *** 14.7-0ubuntu0.22.04.1 500
+        500 http://apt.pop-os.org/ubuntu jammy-security/main amd64 Packages
+        500 http://apt.pop-os.org/ubuntu jammy-updates/main amd64 Packages
+        100 /var/lib/dpkg/status
+     14.2-1ubuntu1 500
+        500 http://apt.pop-os.org/ubuntu jammy/main amd64 Packages
+```
+
+In the above output, the `Candidate` value is `14.7-0ubuntu0.22.04.1`. 
+Using this information, construct the library install command.
+
+```shell
+    sudo apt-get install libpq5=14.7-0ubuntu0.22.04.1 && sudo apt-get install libpq-dev
+```
+
+When the installation process completes, you are prepared to install libpqxx.
 
 Quick start
 -----------
 
-If you just want to get it built and installed quickly, try:
+If you just want to get it built and installed quickly, try the following:
 
 ```shell
-    ./configure
+    cd ~/Downloads
+    git clone https://github.com/jtv/libpqxx.git && cd libpqxx
+    ./configure --disable-shared
```

Comment:
Good idea.  I've recommended it in other places, but this really is where it belongs.

Reply:
Glad at least one thing is useful. :)

Reply:
I'm working on a rewrite of all this right now. Feel free to keep suggesting changes.

Reply:
Oo excellent.  Thanks for sticking with this.